packages/data-models/deployment.model.ts

```diff
@@ -20,8 +22,14 @@ export interface IDeploymentRuntimeConfig {
      * */
     endpoint?: string;
   };
+  analytics: {
+    enabled: boolean;
+    provider: "matomo";
+    endpoint: string;
+    siteId: number;
+  };
```

Comment:
new deployment config available (previously hardcoded into environment)

Reply:
looks good, shall we add some example values to the debug deployment config, if you haven't done so already?